radar_patient_check/database.py

```diff
@@ -1,8 +1,11 @@
-import os
+from sqlmodel import create_engine, Session
+from .config import settings
 
-from dotenv import find_dotenv, load_dotenv
-from sqlmodel import create_engine
 
-load_dotenv(find_dotenv())
+def get_session():
```

Comment:
Considered good practice to use dependencies in fastapi, see https://sqlmodel.tiangolo.com/tutorial/fastapi/session-with-dependency/#create-a-fastapi-dependency